test/polygon/asset/asset.test.ts

```diff
@@ -204,4 +224,116 @@ describe('PolygonAsset.sol', function () {
       expect(balance2).to.be.equal(3);
     });
   });
+
+  describe('Asset <> PolygonAsset: Transfer', function () {
+    it('can transfer L1 minted assets: L1 to L2', async function () {
+      const mainnet = await setupMainnetAsset();
+      const polygon = await setupPolygonAsset();
+      const tokenId = await mainnet.mintAsset(mainnet.users[0].address, 20);
+
+      const balance = await mainnet.Asset['balanceOf(address,uint256)'](
+        mainnet.users[0].address,
+        tokenId
+      );
+
+      // @review - why does ownerOf(tokenId) throw "NFT_!EXIST"?
+      // const mainnet_owner = await mainnet.Asset['ownerOf(uint256)'](tokenId);
```

Comment:
I think this is because the asset is not an ERC721, for that it should have a supply = 1